doc/userguide/install.rst

```diff
@@ -16,6 +16,9 @@ Source
 
 Installing from the source distribution files gives the most control over the Suricata installation.
 
+The Suricata source distribution files should be verified before building
+the source, see :doc:`verifying-source-files`.
+
 Basic steps::
 
     tar xzvf suricata-6.0.0.tar.gz
```

Comment:
thinking we could replace `6` with `7` here, to be more in theme :P

Reply:
will update for the next pr